appeals/web/src/server/lib/mappers/appeal/appeal.mapper.js

```diff
@@ -0,0 +1,174 @@
+import config from '#environment/config.js';
+import { generateIssueDecisionUrl } from '#appeals/appeal-details/issue-decision/issue-decision.mapper.js';
+import {
+	mapDocumentDownloadUrl,
+	mapVirusCheckStatus
+} from '#appeals/appeal-documents/appeal-documents.mapper.js';
+import { APPEAL_CASE_STATUS, APPEAL_VIRUS_CHECK_STATUS } from 'pins-data-model';
+import { submaps as hasSubmaps } from './has.js';
+import { submaps as s78Submaps } from './fpa.js';
```

Comment:
Very minor, could this be renamed to `fpaSubmaps`? I'd be happier if these names used the fully-qualified `submapper` rather than just `submap` but it's not the end of the world

Reply:
Yes good spot, I forgot about this naming convention